### PR TITLE
2023 redesign

### PR DESCRIPTION
**Pull request by depatchedmode**
Full redesign of the website. Has been given the green light by Becky and Bruno.

Lighthouse scores have dropped a tad but they are still fairly good:
image

Needs an additional accessibility pass, and we have some more functional and aesthetic improvements to work on, but I think those can be tabled for later.

### CODE REVIEW

File: src/_data/projects.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+require("dotenv").config();
+const Image = require("@11ty/eleventy-img");
+
+var Airtable = require("airtable");
```
Comment (avivash) [inline src/_data/projects.js:4]: not a show stopper, but can we use either `const` or `let` in place of all the `var`s in this file? Can be done after merging

File: src/_data/team.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+require("dotenv").config();
+const Image = require("@11ty/eleventy-img");
+
+var Airtable = require("airtable");
```
Comment (avivash) [inline src/_data/team.js:4]: same note about the `var`s in this file

File: src/_data/team.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                fetchNextPage();
+            }
+        );
+    
+    for (let i = 0; i < allTeamMembers.length; i++) {
+        
+        // cache images
+        // TODO: make it so it wasn't written by a designer
+        let teamMember = allTeamMembers[i];
+        let resizedAvatar = await Image(teamMember.sourceAvatarURL, {
```
Comment (avivash) [inline src/_data/team.js:57]: can be `const`

File: src/_data/todayProjectIDs.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+require("dotenv").config();
+
+var Airtable = require("airtable");
+var base = new Airtable({ apiKey: process.env.AIRTABLE_API_KEY }).base(
```
Comment (avivash) [inline src/_data/todayProjectIDs.js:4]: same note about the `var`s

File: src/_data/tomorrowProjectIDs.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+require("dotenv").config();
+
+var Airtable = require("airtable");
```
Comment (avivash) [inline src/_data/tomorrowProjectIDs.js:3]: same `var`s note

**Comment by avivash** [comment]
@depatchedmode looks like these links are broken(pointing internally instead of externally)

image

image

**Comment by avivash** [comment]
@depatchedmode looks like some of the links in the nav are _kind of_ broken when viewing the site view IPFS https://ipfs.runfission.com/ipns/fission-2023-redesign-staging.fission.app. I don't think it's a deal breaker though because it looks like they're just hard-coded to point to fission.codes, so it should work once this is deployed to prod

**Comment by avivash** [comment]
@depatchedmode looks like there's a merge conflict

**Comment by depatchedmode** [comment]
@avivash I've made the var tweaks and updated the content in Airtable to correct the CTA links.

For the merge conflict, it's because I deleted that file and just moved the newsletter into the footer — that's the only place it exists now. Unsure the best way to resolve that; my GitFu is rusty. Could you handle it?

**Review by avivash — APPROVED** [review]
I think we've covered all the mission critical bits for launch, nice work!

**Pull request merged**